### ISSUE TITLE
Mixed Containers

### ISSUE THREAD

**Issue by mpaiva** [armilab/select#6]
by @inasens - When selecting a product for within a container, will it be 100% of the time only one product/category, meaning, will we ever mix? As in development, this is something that will be hard to go back on.

**Comment by mpaiva** [comment]
As per @berniwar, the system will impose not mixing different product types. If there's a need to mix, we will need to rethink this approach. @berniwar can you comment on this?

**Comment by mpaiva** [comment]
from @berniwar  - I am forcing one quality per order/container as it is something that makes things easier for the factory. Yet it's something that we do and have done for clients(ie one pallet of PVC as a sample/ half one quality/half another. But see the value of having that option if we need it. Originally wanted to make things easier and only do one thickness and that was quickly proven to be a bad decision. The more I think of it, the more interesting it would be if we could mix product later on if needed.

**Comment by berniwar** [comment]
@mpaiva Yes, the priority now one quality per factory, that is the behavior I need(would like) to enforce, **yet there are exceptions** as **when we are doing samples of bundles**(a sample is 1-3 bundles of another product category or quality), its the only way the client can see the product up front. **These are special** cases and **not a default user flow** at this time. **Can only be done via Admin or sales only.** 

Additional Note: As being able to mix the container is something that **if not implemented early in development it will backtrack us almost to zero**, see that it should be implemented to mitigate the risk. We had a similar issue before, and it delayed us significantly.

**Comment by berniwar** [comment]
@inasens @mpaiva Let me know if that is clear and makes sense.
